test/test.hpp

```diff
@@ -0,0 +1,49 @@
+//
+// test/test.hpp
+// PSim
+//
+// Contributors:
+//   Kyle Krol  kpk63@cornell.edu
+//
+// Pathfinder for Autonomous Navigation
+// Space Systems Design Studio
+// Cornell Univeristy
+//
+
+#ifdef PAN_PSIM_TEST_TEST_HPP_
+#error test.hpp should not be included twice
```

Comment:
Why are you erroring here?

Reply:
This code is supposed to be an extension to a single cpp file. If you're including it twice you're doing something wrong lol. It is a little weird for sure. I just hacked this together to get stuff to build and up and running with CI.

Reply:
Yeah idk - I've been looking at this for a while now. I feel weird giving it a standard header guard when the code in it behaves far different from a normal header file (`main` is even defined inside it!)

Reply:
Oh, now I get it. This is actually a really cool setup; it minimizes overhead.